collection_requirements.rst

```diff
@@ -39,7 +39,10 @@ Collection Infrastructure
 =========================
 
 * MUST have a publicly available issue tracker, that does not require a paid level of service to create an account or view issues.
-* MUST use the `Ansible Code of Conduct (CoC) <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ by copying `CODE_OF_CONDUCT.md <https://raw.githubusercontent.com/ansible-collections/.github/main/CODE_OF_CONDUCT.md>`_ into the root of your Git repository
+* Collections MUST have a Code of Conduct (CoC)
```

Comment:
```suggestion
* Collections MUST have a Code of Conduct (CoC)

```
I think the newline is needed for formatting.

Reply:
Bleah, yeah, I changed levels and forgot that I needed to add blank lines when I did that.